saltstack/states/github-actions/runner.sls

```diff
@@ -47,3 +51,5 @@ runner-enable-service:
     - name: actions.runner.tinkerbell.{{ grains.nodename }}.service
     - enable: True
     - reload: True
+
+# TODO: add docker
```

Comment:
I suspect that it would probably be easiest to just add the docker install steps to pulumi/src/github-actions/cloud-init.go